tests/integration/conftest.py

```diff
@@ -224,23 +225,46 @@ def c2r_config(os_release):
     return ConfigUtils(config_path)
 
 
+@pytest.fixture()
+def config_at():
+    """Factory of the ConfigUtils object.
+
+    Created for simplicity injecting it into your testing unit (no need to import).
+
+    Example:
+    >>> with config_at(Path("/etc/system-release")).replace_line(
+    >>>     "release .+",
+    >>>     f"release {os_release.version[0]}.1.1111",
+    >>> ):
+    """
+
+    def factory(path: Path) -> ConfigUtils:
+        return ConfigUtils(path)
+
+    return factory
+
+
 @pytest.fixture
-def get_system_release(shell):
+def system_release(shell):
     """
+    This function to be used whenever we need live information about system release.
+    E.g. after conversion system release check.
+    Otherwise, use hardcoded SYSTEM_RELEASE envar from /plans/main.fmf
```

Comment:
@bocekm and @danmyway this name might not get confusing if we have in the same file the `system_release` fixture, and the `SYSTEM_RELEASE` that comes from `os.environ`? What would be a good naming to differentiate both of them in a file? 

Maybe, `SYSTEM_RELEASE_ENV`?

Reply:
that looks good